css_tasks/task11/js/accordion.js

```diff
@@ -0,0 +1,47 @@
+import {elements} from './elements.js';
+export function accordion() {
+  elements.then((data) => {
+    const {service, image} = data;
+    service.addEventListener('click', (e) => handleAccordion(e.target));
+    const handleAccordion = (item) => {
```

Comment:
item is not good name for variable in function if it is not function to iterate array